packages/kbn-apm-synthtrace/src/cli/utils/ssl.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import * as Fs from 'fs';
+import { CA_CERT_PATH } from '@kbn/dev-utils';
+import https from 'https';
+
+export function getFetchAgent(url: string) {
+  const isHTTPS = new URL(url).protocol === 'https:';
+  const isLocalhost = new URL(url).hostname === 'localhost';
+  return isHTTPS && isLocalhost ? new https.Agent({ rejectUnauthorized: false }) : undefined;
```

Comment:
Would it be possible to specify the `ca` and set `rejectUnauthorized: true` the same way that we do in `getEsClientTlsSettings`? Or is there an advantage to doing it this way?

Reply:
I think I tried this back when I did the update for the data forge script and wasn't able to get it to work. I found that the kibana dev environment itself uses the same under the hood (see `packages/kbn-cli-dev-mode/src/base_path_proxy/http1.ts` for example).

Reply:
Interesting, the proxy seems to supply the cert but still sets `reject` to `false` :shrug: thanks for the explanation